src/Calculator/Button.js

```diff
@@ -0,0 +1,9 @@
+import './Button.css';
+/* eslint-disable react/prop-types */
+/* eslint-disable react/button-has-type */
+
+const Button = ({ className, value }) => (
+  <button className={className}>{value}</button>
+);
+
+export default Button;
```

Comment:
[Optional] As part of the project requirement, Kindly create a directory called `components`, then you can move your `Calculator` folder with the calculator files inside the components directory. 👍 


> ![image](https://user-images.githubusercontent.com/66112760/183757715-b8a7648f-b483-49f5-8f63-3c00771b431c.png)